staging/src/k8s.io/client-go/util/cert/csr_test.go

```diff
@@ -49,7 +49,7 @@ func TestMakeCSR(t *testing.T) {
 	}
 	csrBlock, rest := pem.Decode(csrPEM)
 	if csrBlock == nil {
-		t.Error("Unable to decode MakeCSR result.")
```

Comment:
t.Error -> t.Fatal? why?

Reply:
if csrBlock is nil, a nill pointer will be referenced later, causing panic.
```
	if csrBlock == nil {
		t.Fatal("Unable to decode MakeCSR result.")
	}
	if len(rest) != 0 {
		t.Error("Found more than one PEM encoded block in the result.")
	}
	if csrBlock.Type != CertificateRequestBlockType {
		t.Errorf("Found block type %q, wanted 'CERTIFICATE REQUEST'", csrBlock.Type)
	}
```

Reply:
yes . so t.Error is ok.